### PR TITLE
Cristina create event

### PR DESCRIPTION
**Pull request by enonnai**
Please find in this pull request:
- change of root to redirect to index once signed up/in
- test for sign up with web helper
- log out button & test
- log in link & test
-  'Add your Event' link to index
- web helper for create event and test

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
-  before_action :authenticate_user!
+  #before_action :authenticate_user!
```
Comment (j311y) [inline app/controllers/application_controller.rb]: Delete this instead of commenting

**Review by j311y — CHANGES_REQUESTED** [review]
Just delete the commented out before_action then :shipit:

**Review by dvidgry — APPROVED** [review]
:shipit:

File: spec/web_helpers.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+def sign_up(email = "test@gmail.com")
```
Comment (axcochrane) [inline spec/web_helpers.rb]: perhaps change to test@example.com so that we don't send any emails to a real gmail address

**Review by j311y — APPROVED** [review]

**Review by BenNoonan1991 — APPROVED** [review]

**Review by axcochrane — APPROVED** [review]
:shipit:

**Comment by enonnai** [comment]
Thanks for the feedback guys!!

**Pull request merged**